All design work done for 2024 rollover [DO NOT MERGE] (#1251)
DFE-Digital/bat-design-history: app/posts/publish-teacher-training-courses/2024-06-21-rollover-2024-changes.md

Reviewer: I wonder if it's worth having Publish in the post title specifically? Just because there will be a separate post about rollover changes and rollover comms? And when I was looking in the design history yesterday there were some mad post titles like 'Sprint 2' which is really helpful! So maybe 'Design changes to Publish teacher training courses' or something like that, and the description 'All amends that went live for the 2024 rollover' is fine?

Reviewer 2: Typically we don't put the service name in the post title as they are commonly viewed in the context of the service.

---
title: Rollover 2024 changes
description: All amends that went live for the 2024 Rollover
date: 2024-06-21
tags:
  - applications
  - courses
  - markdown

>   - copy courses

Reviewer 2: ```suggestion
  - copy courses
  - rollover
```

---

> A number of changes to the prototype had been made over the past year following best practise and research recommendations, which had not made it on to the live service.

Reviewer: All the way through it's best practice not practise

> A number of changes to the prototype had been made over the past year following best practise and research recommendations, which had not made it on to the live service.

Reviewer 2: ```suggestion
Several changes to the prototype had been made over the past year following best practices and research recommendations, but they had not made it to the live service.
```

Additionaly, we moved the Markdown Guidance from the right-hand column to be in-situ above the text area so it's easier to refer too

![Screenshot of course fees page](course-fees.png "Course fees page")
![Screenshot of course length](course-length.png "Course length page")
![Screenshot of financial support](financial-support.png "Financial support page")

### 2. Adding in Markdown Guidance

Some pages with text areas were missing the Markdown guidance, we added these in to the following pages:

- About the Accredited Provider
- Training with your organisation
- Training with disabilities and other needs

![Screenshot of accredited providers](accredited-providers.png "Accredited providers page")
![Screenshot of about organisation](about-org.png "About organisation page")

### 3. Removing Personal Qualities and Other Requirements

Whilst originally we were looking to break these sections down into seperate pages, further analysis showed that the content in the personal qualities and other requirements fields were commonly repeated across all courses - making them less useful to candidates who are looking for things that make courses different from one another.

> We already surface information which is very similar to the content that providers are entering in personal qualities on the screen where candidates enter their personal statement. We feel that this is the most relevant place for this content in the journey.

Reviewer: Can add a note about Other qualities and where we've relocated that information:

The content entered in the Other qualities field covered the mandatory checks before an offer of training can be accepted (DBS and fitness to train to teach checks). We already have this information on the start pages for Find and Apply and in the safeguarding declaration in Apply. We will add it to the email candidates receive when they get an offer to join a course, and to the Get Into Teaching website so that candidates can see this information throughout their journey.

User research with candidates also told us that some find it difficult to understand what a DBS check is, so we have explained what it means in the content.

We aren’t dropping columns because we’ll need to see the data for providers to see for courses in the past.

It is going to sit behind a feature flag

### 4. Changes to Course Details

As we had made some changes to title changes on other pages - this information on course details page was also updated.

There had also been some work done on the prototype which never went live, including:

- Removing the truncation so that all copy is visible
- Using the 'Tab' design for the Basic Details / Description links
- Removal of Personal Qualities
- Removal of Other Qualities

![Screenshot of courses](courses.png "Courses page")

### 5. SEND checkbox to radio buttons

To match how the SEND option appears in the edit section, we changed the SEND question from a checkbox to radio buttons

![Screenshot of send option](send.png "SEND option")

### 6. Full Time or Part Time radio buttons to checkboxes

This has been changed from radio buttons to checkboxes to follow best practises. Users can now chose either or both.

![Screenshot of full time or part time option](full-part-time.png "Full time or part time option")

### 7. Add a Second Language validation

> Some validation has been added so that users cannot select the same language as a secondary option that they already selected as a primary option

Reviewer: I wonder if primary and secondary are confusing here only because of the context of education! Maybe in this case just use first and second (even though primary and secondary are correct)?
